bluemira/base/parameter_frame/_parameter.py

```diff
@@ -133,8 +133,12 @@ def value(self) -> ParameterValueType:
     def value(self, new_value: ParameterValueType):
         self.set_value(new_value, source="")
 
+    def value_as(self, unit: Union[str, pint.Unit]) -> ParameterValueType:
+        """Return the current value in a given unit"""
+        return raw_uc(self.value, self.unit, unit)
+
     @property
-    def unit(self) -> str:
+    def unit(self) -> pint.Unit:
```

Comment:
Are we sure we want to expose `pint` in this way? I thought our plan was just to expose strings, rather than objects, and use `pint` under-the-hood.

Reply:
My thoughts were it would give us the flexibility to display it as we feel like (as a string: https://pint.readthedocs.io/en/stable/formatting.html?highlight=format#unit-format-specifications). I should modify this so the return type is a string though..